spacy_llm/tasks/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .entity_linking import EntityLinkingTask, make_entitylinking_task
```

Comment:
For these longer class/function names, I would consider abbreviating to `EL` so we'd have `ELTask`, `make_el_task` etc.

Reply:
Do you think it's that bad? :thinking:  I'm ok either way.

Reply:
No, it's not that bad, and I don't feel strongly 🤷

Reply:
Ok, leaving it as is then.